specs/compilation/statements/pattern_matching_spec.js

```diff
@@ -129,4 +129,22 @@ describe("Compiling patternMatching statements", () => {
       ),
     );
   });
+
+  it("compiles appropriately when using any identifier", () => {
```

Comment:
This situation is not really possible. You represented this:

```
case X of person then _ = B end
```

This is invalid syntax. I think you were trying to do this instead:

```
case X of person(age:_) then A = B end
```

Reply:
it is an error when merging conflicts to other pr, removed.